.github/workflows/style.yml

```
@@ -0,0 +1,100 @@
+  pull_request:
+    branches: [ main, master ]
+    paths-ignore:
+      - 'eventyay_business/locale/**'
+      - 'eventyay_business/static/**'
+
+jobs:
+  isort:
+    name: isort
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)